misc/config_tools/xforms/lib.xsl

```diff
@@ -636,4 +596,17 @@
   </func:function>
   <!-- End of scenario-specific functions-->
 
+  <!-- Board-specific functions-->
+  <func:function name="acrn:get-normalized-closinfo-rdt-res-str">
+    <xsl:variable name="rdt_resource" select="translate(substring-before(substring-after(//CLOS_INFO, 'rdt resources supported:'), 'rdt resource clos max:'), '&#x9;&#xa;&#xd;&#x20;', '')" />
```

Comment:
Please refer to these escaped whitespace characters using the variables (e.g. `newline`, `tab`, etc.) which are defined in lib.xsl. This can greatly improve the readability of this piece.

Reply:
I don't see `whitespaces` defined anywhere. Have I missed anything?

Reply:
> I don't see `whitespaces` defined anywhere. Have I missed anything?

@junjiemao1 Please expand this file "https://github.com/projectacrn/acrn-hypervisor/pull/6526/commits/1525910a57d61dc75ed422e22503507e0b664511#"
and find line 25

@NanlinXie

Reply:
Fine with me then.